frontend/src/stores/chatStore.ts

```diff
@@ -37,6 +52,55 @@ export const useChatStore = defineStore('chats', {
 			}
 		},
 
+		async createNewChannel(createNewChannelForm: CreateNewChannelForm) {
+			try {
+				this.errors.length = 0;
+				const all = (await getRequest('chats')).data;
+				const found = all.find(
+					(chat: Chat_List_Item) => chat.name === createNewChannelForm.name,
+				);
+				if (found !== undefined){
+					this.errors.push('channel name already exists');
+					return;
+				}
+				if (!createNewChannelForm.name ){
+					this.errors.push('Not a valid channel name');
+					return;
+				}
+				if (!createNewChannelForm.users[0]){
+					this.errors.push('you need to assign users to this channel');
+					return;
+				}
```

Comment:
You don't really, right? Technically you yourself will always be in it (as the owner), so if you just want to create a channel with just you, you could...
but 🤷 maybe it's nice to have this message anyway ;P

Reply:
I agree! but if I take this out it looks like something goes wrong in showing the channel?
keep this in? if you know a quick solution I'm fine with it.
I did change the message that the channel needs adding one or more members so the error is more accurate?
but maybe you don't agree with the conditions..